google/ads/google_ads/v2/services/enums.py

```diff
@@ -78,7 +78,10 @@ def __getattribute__(self, name):
                 # If the class attribute has been replaced with something else,
                 # another must have already executed it and created our type.
                 return class_def_src
-            assert class_def_src.startswith('class %s(enum.' % enum_type_name), 'Expected ' + enum_type_name
+            assertion = class_def_src.startswith(
+                'class %s(enum.' % enum_type_name) or class_def_src.startswith(
+                    'class Operator(enum.')
+            assert assertion, 'Expected ' + enum_type_name
```

Comment:
`class_def_src` will be the stringified definition of the enum, for example [here](https://github.com/googleads/google-ads-python/blob/2e11de5f0edbb561b6c186aeef95a5605336723a/google/ads/google_ads/v3/services/enums.py#L163). This assertion checks that the string either starts with `'class AccessRole(enum.'` if, for example, the outer class name is `AccessRoleEnum`. This follows the pattern for the majority of our API enums, (see [here](https://github.com/googleapis/googleapis/blob/master/google/ads/googleads/v3/enums/access_role.proto#L31)), where the outer message is `FooBarEnum` and the inner enum is named `FooBar`. In a few cases enums are defined on Operations, for example [`FeedAttributeOperation`](https://github.com/googleapis/googleapis/blob/bf839ae632e0f263a729569e44be4b38b1c85f9c/google/ads/googleads/v1/resources/feed.proto#L175), and the inner enum is called `Operator` so this assertion also allows the patter `'class Operator(enum.'`

Reply:
This is valuable information so therefore should be in the comments rather than left in this PR.

Reply:
Sounds good I'll add.